backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -264,7 +264,7 @@ def delete_collection(collection_id: str, token_info: dict):
 
 
 @dbconnect
-def update_collection(collection_id: str, body: dict, token_info: dict):
+def update_collection(collection_id: str, body: dict, token_info: dict, keep_links: bool = False):
```

Comment:
Since this is used directly by the portal API we shouldn't put parameters that are not in the portal API. Break this function into its own common function that is used by both curation and portal.